### ISSUE TITLE
Merge baremes-ipp branch into master

### ISSUE THREAD

**Issue by cbenz** [openfisca/openfisca-france#408]
All formulas must reference legislation parameters paths (when they use `legislation_at`) from new [`parameters` directory](https://github.com/openfisca/openfisca-france/tree/baremes-ipp/openfisca_france/parameters).

This issue is blocked by https://github.com/openfisca/openfisca-france/issues/407

Only then we can merge the `baremes-ipp` branch into `master`.

**Comment by cbenz** [comment]
Superseded by #410
